internal/parser/planparserv2/pattern_match.go

```diff
@@ -0,0 +1,89 @@
+package planparserv2
+
+import (
+	"fmt"
+
+	"github.com/milvus-io/milvus/internal/proto/planpb"
+)
+
+// Note: just like PostgreSQL, we just plan to support two wildcard in pattern match: `_` and `%`.
+
+var wildcards = map[byte]struct{}{
+	'_': {},
```

Comment:
we can not support _ yet because it has to match exact character number

Reply:
removed